### ISSUE TITLE
Make GLA (Demo) bikes less fire resistant

### ISSUE THREAD

**Issue by ExiLe-ZH** [TheSuperHackers/GeneralsGamePatch#604]
Seems silly how a DB can drive straight thru a firewall and kill the dragon tank, while normal infantry seems to die much faster to a fire? Or is this false?

Would be fair since China only has a very few counters on the ground (ECM’s and vetted BM’s) which still are far from reliable counters. Every single unit lacks the mobility to evade aswell.

**Comment by Jundiyy** [comment]
Infantry and Bikers have different armour sets, in many cases, Bikers have more resistance, not just to flames but to other damage types as well. I don't think this needs to be changed.

**Comment by ExiLe-ZH** [comment]
Vs other types of damage I don't think anything needs to be changed, it just doesn't make sense how bikers have more fire resistance since they're sitting on their bikes without any additional protection.

Would be good IMO to give China better defensive capabilities vs OP DB's. That wouldn't make China itself OP suddenly. Demo player just has to learn to avoid flamewalls and if China blocks entrances, Demo still got counters like Scud Launchers.

**Comment by Jundiyy** [comment]
I don't know, whilst it surely does help it might make Demo Bikes really bad since Flame Walls are actually good vs them, if a DB drives through the middle, it loses a lot of hp, it just needs another unit there to be sure to take it down, you just need to have one other unit there and it should die before getting close enough, 2 Flame Walls for example. If you drive through the edges of the fire, then you will get through quite easily which is fair anyway.
The problem would be, Flame Walls alone would end up killing all DBs, making them useless. This already happens if you stack Flame Walls. You can send 1 DB or 10, all will die.

**Comment by ExiLe-ZH** [comment]
I disagree it's good protection, in my experience 1 DB always just kills a Dragon, even when going straight through Black Napalm. Sure you can beef it up by having more units giving cover but it scales badly, you need alot more units to protect an area than the amount of DB's needed. And even then you can flank easily with DB's and kill stuff anyway, especially on more open maps.

Don't agree either that is a problem, it's the whole point of the idea actually. The player just has to learn you can't drive through firewalls, just like you learn not to drive thru mines.

**Comment by commy2** [comment]
Does death by fire make them explode?

**Comment by EnigmaZift** [comment]
Infantry take 150% from Flame type damage. Bikes take 50% from Flame type damage. So they are 3x more resistant. Just so you guys know. It is quite a bit.

**Comment by ExiLe-ZH** [comment]
> 
> 
> Does death by fire make them explode?

Yep, is why it can still kill the Dragon.

> 
> 
> Infantry take 150% from Flame type damage. Bikes take 50% from Flame type damage. So they are 3x more resistant. Just so you guys know. It is quite a bit.

That is a lot indeed.

While it should be 150% aswell in all honesty, I'd like to try 100% too.
